scripts/ipfs/install

```diff
@@ -31,3 +31,22 @@ sudo systemctl restart nginx.service
 
 # add cjdns bootstrap address
 ipfs bootstrap add /ip6/fc6e:691e:dfaa:b992:a10a:7b49:5a1a:5e09/tcp/4001/ipfs/QmU6NeD2Uu34WKest1NZGvGmScLhN1zVo66K35GeE6Jft2
```

Comment:
Which node is this? If it's a central node on the Internet, can we have this as a var defined at top of file and add comment describing?

Reply:
This is heavy metals node that allows us to discover other hyperbole nodes on hyperbole only connections

Reply:
@benhylau I think it's one of @darkdrgn2k 's servers, the one which also serves as hyperboria DNS server.

Reply:
Oh i guess so i though heavy metal ran that one.  Guess not. Added comment